grow/pods/pods.py

```diff
@@ -40,15 +41,14 @@ class PodDoesNotExistError(Error, IOError):
     pass
 
 
-class PodSpecParseError(Error):
-    pass
-
-
 # TODO(jeremydw): A handful of the properties of "pod" should be moved to the
 # "podspec" class.
 
 class Pod(object):
 
+    FILE_PODCACHE = 'podcache.yaml'
```

Comment:
To avoid confusion with `podspec.yaml` I'd remove the `pod` from the name. It's been really nice to autocomplete `po-<TAB>` so far and I don't want to draw too much attention to this file since people won't be regularly interacting with it. Thoughts?